test/helper/Puppeteer_test.js

```diff
@@ -26,6 +26,12 @@ describe('Puppeteer', function () {
       url: siteUrl,
       windowSize: '500x700',
       show: false,
+      chrome: {
```

Comment:
To resolve the merge conflict, this will also need the waitForTimeouts from master, like so:

```
chrome: {
        args: [
          '--no-sandbox',
          '--disable-setuid-sandbox',
        ],
      waitForTimeout: 2000,
      waitForAction: 500,
      },
```